src/fromTime.js

```diff
@@ -0,0 +1 @@
+export default time => new Date(time)
```

Comment:
I wonder should we call this function `fromUnixTime` and rename `unixTime` to `toUnixTime`.